### PR TITLE
Add title to popup form

### PR DESCRIPTION
**Pull request by CFarcy**

### CODE REVIEW

File: src/modules/RA/DataLayer/components/tabs/PopupTab/index.js [hunk 1]
```diff
@@ -50,24 +60,32 @@ const PopupTab = () => {
...
+      );
+
+      const lines = contentfields.map(({
         suffix,
         prefix,
         field: { name },
         default: defaultTitle,
       }) => (
-        `${prefix}: {% if ${name}.length > 0} %}{{${name}}} ${suffix}{% else %}${defaultTitle}{% endif %}`
+        `- ${prefix} : {% if ${name} %}{{${name}}} ${suffix}{% else %}${defaultTitle}{% endif %}`
```
Comment (jrmi) [inline src/modules/RA/DataLayer/components/tabs/PopupTab/index.js:76]: 0 values are defined and shouldn't show default message.

Comment (jrmi) [reply src/modules/RA/DataLayer/components/tabs/PopupTab/index.js:76]: Also the prefix is not the line title. the prefix is just before the value. Use the field name instead.

Comment (jrmi) [reply src/modules/RA/DataLayer/components/tabs/PopupTab/index.js:76]: Handle value round.

**Review by jrmi — CHANGES_REQUESTED** [review]

File: src/modules/RA/DataLayer/components/tabs/PopupTab/index.js (hunk 1 above)
Comment (CFarcy) [reply src/modules/RA/DataLayer/components/tabs/PopupTab/index.js]: Done.

File: src/modules/RA/DataLayer/components/tabs/PopupTab/index.js [hunk 2]
```diff
@@ -47,27 +57,49 @@ const PopupTab = () => {
...
         default: defaultTitle,
-      }) => (
-        `${prefix}: {% if ${name}.length > 0} %}{{${name}}} ${suffix}{% else %}${defaultTitle}{% endif %}`
-      ));
+        sourceFieldId,
+      }) => {
+        const { settings: { round } } = fields.find(f => (f.sourceFieldId === sourceFieldId));
+        if (round !== undefined) {
+          return (
+            `- ${label} : ${prefix} `
```
Comment (jrmi) [inline src/modules/RA/DataLayer/components/tabs/PopupTab/index.js:79]: Prefix part should be inside if, like the suffix.

**Review by jrmi — DISMISSED** [review]

File: src/modules/RA/DataLayer/components/tabs/PopupTab/index.js (hunk 2 above)
Comment (CFarcy) [reply src/modules/RA/DataLayer/components/tabs/PopupTab/index.js]: Done.

**Review by jrmi — APPROVED** [review]

**Pull request merged**